practice-one/src/sections/header.html

```diff
@@ -13,4 +13,9 @@
     "text": "Download App"
     }'></include>
   </nav>
+  <button class="navbar-button">
```

Comment:
div inside button seem wrong please check

Reply:
I changed it to span tag , thanks for your contribution